doc/using-nixpkgs.md

```diff
@@ -5,4 +5,5 @@ using/platform-support.chapter.md
 using/configuration.chapter.md
 using/overlays.chapter.md
 using/overrides.chapter.md
+using/organizing.chapter.md
```

Comment:
This change should be discussed and reviewed separately in https://github.com/NixOS/nixpkgs/pull/270696

Reply:
How do I link to nix.dev documentation here? I assume you want these docs on nix.dev.

Reply:
Once merged you can just link to the nix.dev page directly